src/components/autocomplete/js/autocompleteController.js

```diff
@@ -166,6 +167,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
     var wait = parseInt($scope.delay, 10) || 0;
     $attrs.$observe('disabled', function (value) { ctrl.isDisabled = !!value; });
     $attrs.$observe('required', function (value) { ctrl.isRequired = !!value; });
+    $attrs.$observe('readonly', function (value) { ctrl.isReadonly = !!value; });
```

Comment:
@robertmesserle Done - re-enabled test. I will rebase #6803 if this PR is merged.